### PR TITLE
add celery/workers

### PR DESCRIPTION
**Pull request by caelean**
- Refactoring; add support for render postgres in deployed environments in SDK configure
- Rip out deployment CLI code
- redis service
- update redis
- Clean up console during init command
- Configure staticfiles settings for Django on Render
- Fix incorrect attribute access on postgres environment in ensure_ready
- Finish implementing configuration for Render init command (application callable, project name)
- Move render.yaml write to cwd
- Make sure build/start scripts are executable, add redis to render.yaml
- Bump 0.0.9->0.0.10
- Move build, start scripts to root
- Bump 0.0.10->0.0.11
- update environment strategy
- updated redis
- working celery configuration

### CODE REVIEW

File: bridge/framework/base.py [hunk 1]
```diff
@@ -30,26 +30,26 @@ def is_remote(self) -> bool:
 
     def run(self) -> None:
         """Start services."""
-        if self.is_remote():
-            platform = detect_platform()
-        else:
-            platform = Platform.LOCAL
-            self.start_local_services()
+        platform = detect_platform() if self.is_remote() else Platform.LOCAL
         self.configure_services(platform)
```
Comment (caelean) [inline bridge/framework/base.py:34]: need to configure the services before they start for celery to detect the correct settings

File: bridge/framework/base.py [hunk 2]
```diff
@@ -60,12 +60,22 @@ def start_local_redis(self, client: docker.DockerClient) -> None:
         service = RedisService(client=client, config=config)
         service.start()
 
+    @abstractmethod
+    def start_local_worker(self) -> None:
```
Comment (caelean) [inline bridge/framework/base.py:64]: rn `celery.py` is unique to django

File: bridge/framework/django.py [hunk 3]
```diff
@@ -62,16 +64,52 @@ def configure_staticfiles(self, platform: Platform):
...
                 )
 
+    def configure_worker(self, platform: Platform) -> None:
+        environment = build_redis_environment(platform)
+        self.framework_locals["CELERY_BROKER_URL"] = environment.url()
+        self.framework_locals["CELERY_RESULT_BACKEND"] = environment.url()
+
+    def start_local_worker(self) -> None:
+        # Confirm we are in a `runserver` command
+        if sys.argv[1] == "runserver":
```
Comment (caelean) [inline bridge/framework/django.py]: The django management util was printing out a bunch of stuff and wasn't easy to silence, so have a basic check here for now

File: bridge/framework/django.py [hunk 4]
```diff
@@ -62,16 +64,52 @@ def configure_staticfiles(self, platform: Platform):
...
+                    subprocess.Popen(
+                        [
+                            "celery",
+                            "-A",
+                            "bridge.service.celery",
+                            "worker",
+                            "-l",
+                            "INFO",
+                        ],
+                        stdout=subprocess.DEVNULL,
```
Comment (caelean) [inline bridge/framework/django.py:153]: we could think about doing stuff with this output in the future, for now just silencing

**Review by emdoyle — APPROVED** [review]

**Pull request merged**